README.md

```diff
@@ -169,9 +182,20 @@ export const Page = () => {
 | `referrerKey`           | `PublicKey`              | Public key for referral tracking to create a share of the transaction for the referrer. |
 | `partnerLogo`           | `ReactNode`              | React node to display a partner’s logo within the widget interface.                     |
 
-## Upcoming features/improvements
+## 🧩 Peer Dependencies
```

Comment:
@dr497 WDYT about this section? Almost no one checking package.json specifically, so I think it's worth mentioning this kind of info.

Reply:
That's a good point